bech32py/bech32.py

```diff
@@ -83,12 +83,12 @@ def bech32_decode(bech):
         return (None, None, None)
     if not all(x in CHARSET for x in bech[pos+1:]):
         return (None, None, None)
-    hrp = bech[:pos]
+    human_readable_prefix = bech[:pos]
     data = [CHARSET.find(x) for x in bech[pos+1:]]
-    spec = bech32_verify_checksum(hrp, data)
+    spec = bech32_verify_checksum(human_readable_prefix, data)
     if spec is None:
         return (None, None, None)
-    return (hrp, data[:-6], spec)
+    return (human_readable_prefix, data[:-6], spec)
```

Comment:
If this were JavaScript I'd be worried about changing the name of a field on the return object but since it's Python I think we're good? Would appreciate a sanity check right here to make sure everything is okay.

Reply:
yes this is a tuple so only the values are returned, its not like a JS object where `{thing}` is shorthand for `{"thing": thing}` 👍